modules/tinymce/src/plugins/searchreplace/test/ts/browser/SearchReplaceDialogTest.ts

```diff
@@ -94,4 +94,24 @@ describe('browser.tinymce.plugins.searchreplace.SearchReplaceDialogTest', () =>
     findAndAssertFound(editor, 2);
     TinyUiActions.closeDialog(editor);
   });
+
+  // keyboard shortcut
```

Comment:
Please don't place a comment standalone like this... use the `context` block instead if we need to separate tests.

Reply:
https://github.com/tinymce/tinymce/pull/7533/commits/25a47eb7a1a360e960efb0732a6f7fbd84a85990